.circleci/comment.js

```diff
@@ -40,6 +40,19 @@ async function run() {
             break;
           }
         }
+      } else {
+        //If it isn't a PR commit, then we are on main. Create a comment for the test app and docs build
+        await octokit.repos.createCommitComment({
+          owner: 'adobe',
+          repo: 'react-spectrum',
+          commit_sha: process.env.CIRCLE_SHA1,
+          body: `Verdaccio builds:
+      [Test App](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/verdaccio/build/index.html)
+      [Test App Size](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/verdaccio/publish-stats/build-stats.txt)
+      [Publish stats](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/verdaccio/publish-stats/publish.json)
+      [Size diff since last release](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/verdaccio/publish-stats/size-diff.txt)
+      [Docs](https://reactspectrum.blob.core.windows.net/reactspectrum/${process.env.CIRCLE_SHA1}/verdaccio/docs/index.html)`
+        });
```

Comment:
Also I see these weren't added to the below PR commenting logic. So does this mean the comments only go on commits and not on PRs?

Reply:
That is correct, I figured we'd only really want to do the verdaccio builds on commits to main since it is quite time consuming and I'd rather not hold up PR merges

Reply:
Ah I see. Maybe we can chat about this with the team. I wonder if we might want to know before merge whether we broke the prod docs build or something. But we can see how it goes.